scripts/wallet.js

```diff
@@ -1230,7 +1229,7 @@ export class Wallet {
         }
         const txs = await db.getTxs();
         for (const tx of txs) {
-            this.addTransaction(tx, true);
+            await this.addTransaction(tx, true);
```

Comment:
This missing await is what has been causing issues with db loading.
Changed this to
```suggestion
          (async () => {
             await sleep(Math.random() * 1000)
             this.addTransaction(tx, true);
          })()
```
To simulate weird promise scheduling
And i've had the same issue that's been described in tickets.